yarn.lock

```diff
@@ -3561,11 +3561,11 @@ copy-props@^2.0.1:
     is-plain-object "^2.0.1"
 
 core-js-compat@^3.1.1:
-  version "3.2.1"
-  resolved "https://registry.yarnpkg.com/core-js-compat/-/core-js-compat-3.2.1.tgz#0cbdbc2e386e8e00d3b85dc81c848effec5b8150"
-  integrity sha512-MwPZle5CF9dEaMYdDeWm73ao/IflDH+FjeJCWEADcEgFSE9TLimFKwJsfmkwzI8eC0Aj0mgvMDjeQjrElkz4/A==
+  version "3.4.6"
```

Comment:
🤔I guess here is the culprit: For some reason (CDN cache?) 3.4.7 is not installed.

Reply:
Could you re-run yarn upgrade so the lockfile can be updated? (Also Circle CI cache can be thus reset again)